torch/nested/_internal/ops.py

```diff
@@ -2797,3 +2799,41 @@ def _nested_get_jagged_dummy(func, *args, **kwargs):
     aten.impl("_nested_get_jagged_metadata", _nested_get_jagged_dummy, "CPU")
     aten.impl("_nested_get_jagged_metadata", _nested_get_jagged_dummy, "CUDA")
     aten.impl("_nested_get_jagged_metadata", _nested_get_jagged_dummy, "Meta")
+
+
+@register_jagged_func(
+    [
+        torch.ops.aten.zeros.default,
+        torch.ops.aten.full.default,
+        torch.ops.aten.ones.default,
+    ],
+    "size: any, dtype: any?, layout: any?, device: any?, pin_memory: any?",
+)
+def jagged_factory(func, *args, **kwargs):
```

Comment:
You can spec args and kwargs using ParamSpec if you want and have func be a callable